Source/Engine/Audio/AudioSource.cpp

```diff
@@ -583,3 +588,11 @@ void AudioSource::BeginPlay(SceneBeginData* data)
             SetTime(GetStartTime());
     }
 }
+
+void AudioSource::EndPlay()
+{
+    Actor::EndPlay();
+
+    ClipStarted.UnbindAll();
```

Comment:
While this might be useful to unbind events automatically, I'm not sure if that's a good call .
For object pooling it could help but if someone is using that event then shuld unregister on end manually, just like for other events in the engine.

What do you think?